src/options-pages/libraries.cpp

```diff
@@ -0,0 +1,79 @@
+/****************************************************************************
+**
+** Copyright (C) 2017 N7 Mobile sp. z o. o.
+** Contact: http://n7mobile.pl/Space
+**
+** This file is part of ASN.1/ACN Plugin for QtCreator.
+**
+** Plugin was developed under a programme and funded by
+** European Space Agency.
+**
+** This Plugin is free software: you can redistribute it and/or modify
+** it under the terms of the GNU General Public License as published by
+** the Free Software Foundation, either version 3 of the License, or
+** (at your option) any later version.
+**
+** This Plugin is distributed in the hope that it will be useful,
+** but WITHOUT ANY WARRANTY; without even the implied warranty of
+** MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+** GNU General Public License for more details.
+**
+** You should have received a copy of the GNU General Public License
+** along with this program.  If not, see <http://www.gnu.org/licenses/>.
+**
+****************************************************************************/
+#include "libraries.h"
+
+#include <QWidget>
+
+#include <asn1acnconstants.h>
+#include <tr.h>
+
+#include "librarieswidget.h"
+
+using namespace Asn1Acn::Internal;
+using namespace Asn1Acn::Internal::OptionsPages;
+
+Libraries::Libraries(Settings::LibrariesPtr settings)
+    : m_settings(settings),
+      m_widget(nullptr)
+{
+    setId(Constants::LIBRARIES_SETTINGS_ID);
+    setDisplayName(Tr::tr("Libraries"));
+    setCategory(Constants::SETTINGS_CATEGORY);
+    setDisplayCategory(Tr::tr(Constants::SETTINGS_CATEGORY_DISPLAY));
+    setCategoryIcon(Utils::Icon(Constants::OPTIONS_CATEGORY_ICON));
+}
+
+bool Libraries::matches(const QString &searchKeyWord) const
+{
+    const QStringList keywords { "asn1", "asn.1", "acn", "libraries", "components" };
+    for (const auto& keyword : keywords)
+        if (keyword.contains(searchKeyWord, Qt::CaseInsensitive))
+            return true;
+    return Core::IOptionsPage::matches(searchKeyWord);
+}
+
+QWidget* Libraries::widget()
```

Comment:
Should be QWidget *Libraries::widget()

Reply:
true :)

it's ctrl+cv from other file in that folder, so I'll fix them all